tests/phpunit/class-wc-stripe-checkout-sessions-ajax-handler-test.php

```
@@ -103,6 +114,248 @@ public function test_create_checkout_session(
 		$this->assertEquals( (object) $expected_response, $response );
 	}
 
+	/**
+	 * Checkout Sessions should send one line item whose amount matches the full cart total (Stripe minor units).
+	 */
+	public function test_create_checkout_session_sends_single_line_item_matching_cart_total(): void {
+		Ajax_Test_Helper::init_hooks();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'Ajax_Test_Helper' in method 'test_create_checkout_session_sends_single_line_item_matching_cart_total'. (undefined)

(StaticAccess)